Tickets/SITCOM-1637

kpmort97 opened the pull request:
I migrated the ATDome Fails to Arrive in Position page on Confluence over to obs-ops. Please look it over and provide comments as I am not familiar with the procedures in the document or if there needs to be updates. Thank you!

Garavena89 reviewed (approved):
Nice work, Kriss! Thanks for moving and improving this document.
I just added a couple of minor comments on Procedure A, and the Jupyter notebook section looks good to me. Perhaps, this might be a good opportunity to add a link with details on accessing the USDF Jupyter notebook for newcomers, but I leave that totally up to you, because it is also unlikely we use this again in the near future.
Thanks!

kpmort97 commented on AuxTel/Troubleshooting/ATCS/ATDome-Fails-to-Arrive-In-Position.rst:
```diff
@@ -0,0 +1,219 @@
+
+1. State cycle ATDome through ``STANDBY`` and back to ``ENABLE`` from the ASummaryState. 
+   The transition must be quick enough so the dome shutter doesn´t start closing and the recovery 
+   is faster; if it does close, the next step should deal with the shutter opening again.
```
I'll add an additional note about accessing the scripts as alternative solutions. That way both options will be available in the same step.

kpmort97 commented on AuxTel/Troubleshooting/ATCS/ATDome-Fails-to-Arrive-In-Position.rst:
```diff
@@ -0,0 +1,219 @@
+--------------------------------
+
+1. State cycle ATDome through ``STANDBY`` and back to ``ENABLE`` from the ASummaryState. 
+   The transition must be quick enough so the dome shutter doesn´t start closing and the recovery 
```
Done!

kpmort97 commented:
> Nice work, Kriss! Thanks for moving and improving this document. I just added a couple of minor comments on Procedure A, and the Jupyter notebook section looks good to me. Perhaps, this might be a good opportunity to add a link with details on accessing the USDF Jupyter notebook for newcomers, but I leave that totally up to you, because it is also unlikely we use this again in the near future. Thanks!

Hey @Garavena89 . I think I added the necessary steps to access the jupyter notebooks using Nublado summit instance. As I have not accessed the notebooks before, would these steps be sufficient enough to follow?

edennihy commented on AuxTel/Troubleshooting/ATCS/ATDome-Fails-to-Arrive-In-Position.rst:
```diff
@@ -0,0 +1,227 @@
+       
+.. _ATDome-Position-Failure-Procedure-Jupyter:
+
+Procedure B: Recovery Using Jupyter Notebook 
```
I recommend we deprecate the recovery using a Jupyter Notebook. While it is still supported, we no longer train on using Jupyter notebooks to control the system, and the LOVE system is mature enough to handle any recovery procedures. Plus, we don't cover the notebook recovery of many of our troubleshooting procedures, so I think this is out of place.

kpmort97 replied:
I agree. I will remove the section.

edennihy commented on AuxTel/Troubleshooting/ATCS/ATDome-Fails-to-Arrive-In-Position.rst:
```diff
@@ -0,0 +1,227 @@
+
+.. note::
+
+    One can alternatively state cycle ATDome using either the ``open_dome.py`` or ``set_summary_state.py``
```
The open_dome.py script does not state cycle the dome, but it can be used to re-open them dome if it started closing. However, it will not re-enable dome following, so I think we should always recommend running the prepare_for/onsky script.

kpmort97 replied:
I will omit the open_dome.py part. Luckily, prepare_for/onsky is in the next step, so just having the set_summary_state in the note should be clear enough.

edennihy commented on AuxTel/Troubleshooting/ATCS/ATDome-Fails-to-Arrive-In-Position.rst:
```diff
@@ -0,0 +1,227 @@
+    One can alternatively state cycle ATDome using either the ``open_dome.py`` or ``set_summary_state.py``
+    scripts found in the :ref:`Common AuxTel Scripts <Common-LOVE-SAL-Scripts-and-Configurations>` page.
+
+2. Run ``auxtel/prepare_for/onsky.py`` and move it to the beginning of the ATQueue. 
```
I think we need to have an intermediate step here to run the enable_dome_following script. If the user state cycles the ATDome CSC such that it does not close, then if they run the enable_dome_following script they will not need to run prepare_for/onsky. 

Running the prepare_for/onsky script is a totally valid way to recover, it just takes longer.

kpmort97 replied:
Added!

edennihy reviewed (approved):
A few comments to consider, otherwise approved.
